wiki/Client/Beatmap_editor/AiMod/en.md

```diff
@@ -25,139 +25,143 @@ Development of AiMod usually lags behind the standards held by the mapping commu
+| This mapset needs at least 1 Easy/Normal diff. | *Self-explanatory* | Create an Easy/Normal difficulty for the set. | This warning is based on an ancient ranking criteria rule which has since rendered obsolete. Thus, it can safely be ignored as long as the map follows the [current ranking criteria's spread requirement](/wiki/Ranking_Criteria#beatmap). |
+| This mapset cannot have an Insane diff without a Hard diff. | *Self-explanatory* | Create a Hard difficulty for the set. | This warning is based on an ancient ranking criteria rule which has since rendered obsolete. Thus, it can safely be ignored as long as the map follows the [current ranking criteria's spread requirement](/wiki/Ranking_Criteria#beatmap). |
+| This mapset is missing diff: {0} | *Self-explanatory* | Create the difficulty in question to fill in the difficulty gap. | This warning is based on an ancient ranking criteria rule which has since rendered obsolete. Thus, it can safely be ignored as long as the map follows the [current ranking criteria's spread requirement](/wiki/Ranking_Criteria#beatmap). |
+| This mapset needs at least 2 standard diffs. | *Self-explanatory* | Create a second difficulty for the set. | [](/wiki/shared/mode/osu.png) osu! maps only. This warning is based on an ancient ranking criteria rule which has since rendered obsolete. Thus, it can safely be ignored as long as the map follows the [current ranking criteria's spread requirement](/wiki/Ranking_Criteria#beatmap). |
+| This mapset needs at least 2 osu!taiko diffs. | *Self-explanatory* | Create a second difficulty for the set. | [](/wiki/shared/mode/taiko.png) osu!taiko maps only. This warning is based on an ancient ranking criteria rule which has since rendered obsolete. Thus, it can safely be ignored as long as the map follows the [current ranking criteria's spread requirement](/wiki/Ranking_Criteria#beatmap). |
+| This mapset needs at least 2 osu!catch diffs. | *Self-explanatory* | Create a second difficulty for the set. | [](/wiki/shared/mode/catch.png) osu!catch maps only. This warning is based on an ancient ranking criteria rule which has since rendered obsolete. Thus, it can safely be ignored as long as the map follows the [current ranking criteria's spread requirement](/wiki/Ranking_Criteria#beatmap). |
+| This mapset needs at least 2 osu!mania diffs. | *Self-explanatory* | Create a second difficulty for the set. | [](/wiki/shared/mode/mania.png) osu!mania maps only. This warning is based on an ancient ranking criteria rule which has since rendered obsolete. Thus, it can safely be ignored as long as the map follows the [current ranking criteria's spread requirement](/wiki/Ranking_Criteria#beatmap). |
```

Comment:
```suggestion
| This mapset needs at least 2 osu!mania diffs. | *Self-explanatory* | Create a second difficulty for the set. | [](/wiki/shared/mode/mania.png) osu!mania maps only. This warning is based on an ancient ranking criteria rule which has since rendered obsolete. Thus, it can safely be ignored as long as the map follows the [current ranking criteria's spread requirement](/wiki/Ranking_Criteria/osu!mania#rules). |
```
osu!mania has different spread rules so should link to the specific page for those

Reply:
yup